tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -1133,6 +1136,45 @@ public function __construct(
 }',
         ];
 
+        yield 'multiple attributes ignored' => [
```

Comment:
I think this would show the intention better:

```suggestion
        yield 'keep attributes as-is' => [
```